templates/fine-tune-llm/training_configs/full_param/llama-2-13b-4k-4xg5_12xlarge.yaml

```diff
@@ -11,6 +11,7 @@ num_checkpoints_to_keep: 1
 output_dir: /mnt/local_storage
 deepspeed:
   config_path: deepspeed_configs/zero_3_llama_2_13b.json
+dataset_size_scaling_factor: 10000
```

Comment:
Can you add a comment so that the users don’t set this to a wrong value?

Reply:
added